Wordle.Api/Wordle.Api/Controllers/ScoreController.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ScoreController(ScoreService scoreService) : ControllerBase
+{
+    [HttpGet("TopScores")]
+    public async Task<Score[]> GetTopScores()
```

Comment:
we tried using an array but ended up moving to a list maybe thinking about that will help you understand things

Reply:
Good suggestion, I will look into it. Thanks